pkg/acquisition/modules/wineventlog/wineventlog.go

```diff
@@ -0,0 +1,59 @@
+//go:build !windows
+
+package wineventlogacquisition
```

Comment:
don't we want to error if someone tries to invoke it on non-windows platform ?

Reply:
This is already done here: https://github.com/crowdsecurity/crowdsec/blob/e18a6286dd0234ce728f26c7707ba31b44d3fd5a/pkg/acquisition/modules/wineventlog/wineventlog.go#L50